rds/rds.go

```diff
@@ -392,10 +390,9 @@ func (r *RDS) GetAncestors(areaCode string) ([]models.AreasAncestors, error) {
 		ancestors = append(ancestors, &rs)
 	}
 
-	var a []models.AreasAncestors
-	for _, data := range ancestors {
-		a = append(a, *data)
+	a := make([]models.AreasAncestors, len(ancestors))
```

Comment:
Why are we doing this at all? We are making a slice of pointers to `AreasAncestors` then looping over this to make a slice of `AreasAncestors` as an otherwise direct copy. Why don't we just make the desired slice in the first place? I.e. change line 380 to  `var ancestors []models.AreasAncestors`, add to it in the loop (removing the `&` in line 390), then simply return it rather than lopping over it entirely again.

Reply:
Good point , Again I was really focussed on fixing the lint issues that I didn't read the logic :) .Not sure why it was implemented like this in the first place. Anyways I have fixed this in update 2.